packages/opentelemetry-sdk-trace-base/test/common/export/ConsoleSpanExporter.test.ts

```diff
@@ -99,4 +99,15 @@ describe('ConsoleSpanExporter', () => {
       });
     });
   });
+
+  describe('force flush', () => {
+    describe('when flushing complete', () => {
+      it('should call an async callback', done => {
+        consoleExporter = new ConsoleSpanExporter();
+        consoleExporter.forceFlush().then(() => {
```

Comment:
Would it be better to just convert this test function to be an async function and await the returned promise?